libsrc/target/aquarius/stdio/generic_console.asm

```diff
@@ -57,3 +62,26 @@ generic_console_scrollup:
     bit     2,(hl)
     jp      nz,scrollup_BITMAP
     ret
+
+
+    SECTION code_crt_init
```

Comment:
Would it be better to move this code to one of the crt as files surrounded by IF __HAVE_GENCON and __AQUARIUSPLUS__ so the code isn't even included to save a few bytes and not do the runtime check for CLIB_AQUARIUS_PLUS

Reply:
That would work

Reply:
That me add this change to the PR.